src/views/onboarding/login/styles.scss

```diff
@@ -0,0 +1,102 @@
+@import "src/assets/styles/global/main";
+
+.p-login {
+  height: 100vh;
```

Comment:
It can stay for now, but we want it on onboarding css since same thing should apply to registration page